UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeUtilities.java

```diff
@@ -1684,16 +1684,17 @@ class PropertyAssignment {
                 if (property == null) {
                     continue;
                 }
-                String value = property.getValue(codePoint);
+                ArrayList<String> values = new ArrayList<>();
+                for (String value : property.getValues(codePoint)) {
+                    values.add(value);
+                }
```

Comment:
optional: looks like it could use a Stream
- if getValues() is only Iterable: `property.getValues(codePoint).forEach(values::add);`
- if getValues() is a Collection: `List<String> values = property.getValues(codePoint).stream().collect(Collectors.toList());`